### ISSUE TITLE
Font files with static `FILL` property

### ISSUE THREAD

**Issue by boywithkeyboard** [marella/material-symbols#21]
Do you mind adding another file to the `@material-symbols/font-...` packages that has a static `FILL` option? This should reduce the file size by around 30%.

(follow-up to #14)

**Comment by boywithkeyboard** [comment]
@marella, what do you think?

**Comment by marella** [comment]
Hi, I would prefer not to add more variations as it would triple the number of files.

**Comment by boywithkeyboard** [comment]
@marella, I mean all other properties are already available as a static version, so why the `FILL` property not?

**Comment by marella** [comment]
In `@material-symbols/font-400` and other packages, I used fixed values for `opsz` and `GRAD` as they don't change the appearance of icons much. But `FILL` changes the icon appearance completely and is something users will want to customize so I kept it variable.

By adding separate files for each `wght`, it reduced file size from ~3MB to ~300KB which is a huge improvement but also increased number of files 7 times. Now if I also add `FILL` it will increase number of files 14 times but the reduction in file size is not that large, so I didn't add them.

If you really want to reduce the file size further, it is better to use SVGs [`@material-symbols/svg-400`](https://www.npmjs.com/package/@material-symbols/svg-400) which will significantly reduce the build size depending on the number of icons you use. Most modern frameworks support in-lining SVGs, so they can be styled using CSS like font icons.
